core/src/main/scala/topgun/core/CallSite.scala

```diff
@@ -22,12 +22,12 @@ object CallSite {
   private val IGNORABLE_TOP_FRAME = 2
 
   private val all = new mutable.HashMap[CallSite, CallSite]()
-  def apply(packageName: String, className: String, methodName: String, desc: String, line: Int): CallSite = {
-    val site = new CallSite(packageName.intern, className.intern, methodName.intern, desc.intern, line)
+  def apply(packageName: String, className: String, methodName: String, isNative: Boolean, desc: String, line: Int): CallSite = {
+    val site = new CallSite(packageName.intern, className.intern, methodName.intern, isNative, desc.intern, line)
```

Comment:
interning should occur outside this call. There is no point i n the outer class holding on to the non-interned names